composer.json

```diff
@@ -3,11 +3,11 @@
   "description": "PHP API client for the Alma payments API",
   "type": "library",
   "require-dev": {
-    "phpcompatibility/php-compatibility": "^9.0",
-    "squizlabs/php_codesniffer": "^3.3",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
+    "phpcompatibility/php-compatibility": "^9.0",
+    "phpunit/phpunit": "*",
```

Comment:
@webaaz , pourquoi on ne demande plus phpunit8 en particulier ?

Reply:
Tests should work on PHP5.6 or newer, but latest versions of PHPUnit need at least php7.2.
I have set a version of PHP, so our IDE can help us to code on PHP5.6.